web/src/components/ChatMessage.tsx

```diff
@@ -51,15 +52,21 @@ type ChatMessageProps = { message: ChatMessageModel; showIcon: boolean }
 
 const ChatMessage = ({ message, showIcon }: ChatMessageProps): ReactElement => {
   // TODO 2799 Check if Remote content is really needed here or how external links will be delivered via api
-  const { icons } = buildConfig()
   const navigate = useNavigate()
   const { t } = useTranslation('chat')
-  const { body, userIsAuthor } = message
+  const { body, userIsAuthor, isAutomaticAnswer } = message
+
+  const getIcon = (userIsAuthor: boolean, isAutomaticAnswer: boolean): ReactElement => {
+    if (userIsAuthor) {
+      return <Circle>{t('user')}</Circle>
+    }
+    const icon = isAutomaticAnswer ? ChatbotBot : ChatbotPerson
+    return <Icon src={icon} />
+  }
```

Comment:
:upside_down_face: Optional: Extract to own component (perhaps still in the same file as it is not dependent on any props.